twarep/electrifly
# Worked on the weather table, completed it with database implementation

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
- Edited the app.py file to introduce the Weather Data for Selected Flights Table to the Data Analysis screen,
- Added single-select functionality to the Weather Data for Selected Flights graph,
- Queryed the flight data and weather data to output the time, temperature, wind speed, wind direction, visibility

## How I Did it?
- Created a file called flight_querying.py to query the flights
- Created a file called weather_querying.py to query the relevant weather details

## How to Test?
Testing is simple. Just pull the changes. Then run the app.py script. Once Shiny is running, open the web page in a new browser and see if you can view the following table under "Data Analysis": Note - make sure to select multiple flight dates.
![image](https://github.com/twarep/electrifly/assets/64234627/be4af12e-5b67-48bf-a247-d36dd33e8978)
